pylint_odoo/test_repo/broken_module/model_view_odoo2.xml

```diff
@@ -85,5 +85,25 @@
             </field>
         </record>
 
+        <!-- duplicate record field "name" in *2M field-->
+        <record id="view_model_form5" model="ir.ui.view">
+            <field name="name">view.model.form</field>
+            <field name="model">test.model</field>
+            <field name="arch" type="xml">
+                <form string="Test model6">
+                    <field name="picking_ids">
+                        <tree>
+                            <field name="name"/>
+                        </tree>
+                    </field>
+                    <field name="value_ids">
+                        <tree>
+                            <field name="name"/>
+                        </tree>
+                    </field>
+                </form>
+            </field>
+        </record>
+
```

Comment:
@hugho-ad 
Could you add the following test case?
```xml
+        <!-- Double duplicate record field "name" in *2M field-->
 +        <record id="view_model_form5" model="ir.ui.view">
 +            <field name="name">view.model.form</field>
 +            <field name="model">test.model</field>
 +            <field name="arch" type="xml">
 +                <form string="Test model6">
 +                    <field name="picking_ids">
 +                        <tree>
 +                            <field name="name"/>
 +                            <field name="name"/>
 +                        </tree>
 +                    </field>
 +                    <field name="value_ids">
 +                        <tree>
 +                            <field name="name"/>
 +                            <field name="name"/> 
 +                        </tree>
 +                    </field>
 +                </form>
 +            </field>
 +        </record>
```

Here we should have 2 new messages emitted (increase the expected checkers)